Docs/source/Hardware/Boards.rst

```diff
@@ -439,3 +439,183 @@ and the mux switches between all of them.
+        * Why is it necessary: Takes the voltage and temperature inputs and transmits the data to the leader board 
+        * Justification for selection of specific part:  Standard IC for battery management systems in industry    
+        * `Datasheet link <https://www.analog.com/media/en/technical-documentation/data-sheets/LTC6811-1-6811-2.pdf>`__ 
+        * Associated passives/components:  
+            * Switch (If the IC won't be used for an extended period of time, it can be turned off with this switch.)    
+            * Resistors: 806 ohm, 1.2kohm, Two 100 ohm 
+            * Bypass capacitors     
+            * HX1188FNL 
+            * LTC6255     
+            * Regular SPI Connector (1x5 2.54mm Vertical) 
+            * DTEN Configuration Connector (1x3 2.54mm Vertical)     
+            * Iso Configuration Connector (1x3 2.54mm Vertical) 
+    * HX1188FNL     
+        * Description: Single port surface mount magnetics 
+        * Why is it necessary:  Connects to the input and output connectors to the IPA and IMA connections, 
+          respectively, on the LTC6811.    
+        * Justification for selection of specific part:    
+        * `Datasheet link <https://www.mouser.com/datasheet/2/336/H329-1199189.pdf>`__ 
+        * Associated passives/components:  
+            * Two 120 ohm resistors     
+            * LTC6811  
+    * 1 Molex Micro-fit 3.0 1x5 2.54mm Vertical      
+        * Description: Connector for the Regular SPI connection. 
+        * Why is it necessary: Connects to the IPA, IMA, SDI, and SDO connections on the LTC6811.    
+        * Justification for selection of specific part: Standard component.   
+        * Associated passives/components:  
+            * 5.1k pull-up resistor (for SDO)      
+            * LTC6811  
+    * 2 Molex Micro-fit 3.0 1x3 2.54mm Vertical      
+        * Description:  One connector is for Iso configuration and one connector is for DTEN configuration. 
+        * Why is it necessary: Connects to the ISO and DTEN connections on the LTC6811.     
+        * Justification for selection of specific part: Standard component   
+        * Associated passives/components:     
+            * LTC6811                  
+
+
+
+**List of I/O and Connections:**
+    
+    * Leaderboard Input
+        * Input from BPS Leader Board, tells the Minion board when to gather measurments form the batteries. 
+    * LeaderBoard Output 
+        * Outputs to the leaderboard, transmits voltage and temperature readings from the battery modules.
+    * Voltage Connectors 
+        * Input from the battery modules, transmit voltage from each battery module. The IC also uses this 
+          voltages to power itself. 
+    * Minion Shield Power  
+        * 5V output from the LTC6811 to the Minion Shield board. This is used to power the temperature sensors.
+    * Temperature Input
+        * Input from the Minion Shield board, transmits temperature data.
```

Comment:
Isn't this information also on lines 476-486?

Reply:
this part is repeated because within our outline we have to show the different I/O and connections for different parts of the board but this Board only has a main so it has the same I/O and connections as above. for example if you look at the display board its spilt up into Main, can, and power distribution and each one has their own connections. for this board its completely up to you since it only has a main.

Reply:
I went ahead and got rid of this part and did the same with other boards that also only have a main(at least the new ones I added)